### ISSUE TITLE
Hot module chunk is exported instead of main file

### ISSUE THREAD

**Issue by Anahkiasen** [ztoben/assets-webpack-plugin#22]
When using the plugin with hot module replacement, it seems the hot chunks are exported to the manifest instead of the main file which would then require said chunks.

Consider that this is the output of a change during a hot reload session:

```
                               Asset      Size  Chunks             Chunk Names
                              hot.js   1.69 MB       0  [emitted]  main
0.e5624569d7cdf05006fa.hot-update.js   5.71 kB    0, 0  [emitted]  main, main
e5624569d7cdf05006fa.hot-update.json  36 bytes          [emitted]
                             hot.css    422 kB       0  [emitted]  main
```

What should be in the manifest is `hot.js`, but instead it's `0.e5624569d7cdf05006fa.hot-update.js` which, if included as asset, will just throw `Uncaught ReferenceError: webpackHotUpdate is not defined` and crash. Manually including `hot.js` makes hot reloading work as expected.

**Comment by minheq** [comment]
My work-around this is by changing **getFileExt.js** to separate hot-update from the app bundle.

```
if(asset.indexOf('hot-update') > -1){
    return 'hot-update.js';
} else {
    var extIx = asset.lastIndexOf('.');
    return asset.slice(extIx + 1);
}
```

**Comment by sporto** [comment]
I published v 2.2.3 with the above PR
Could you please check if this fixes the issue?

**Comment by Anahkiasen** [comment]
Seems like it does, yes! Thanks a lot
